web-app/src/components/Navigation/index.js

```diff
@@ -1,27 +1,52 @@
-import React from 'react';
+import React, {Component} from 'react';
 import AppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
 import MenuIcon from '@material-ui/icons/Menu';
 import Typography from '@material-ui/core/Typography';
-import { Link } from 'react-router-dom';
+import { Link, useLocation, withRouter } from 'react-router-dom';
 import Button from '@material-ui/core/Button';
+import './navigation.css'
+import '../../index.css'
+import * as LINKS from '../../constants/navlinks';
 
-import * as ROUTES from '../../constants/routes';
+class Navigation extends Component {
 
-const Navigation = () => (
-    <div>
-        <AppBar style={{backgroundColor:"#32a852"}} position="static">
-            <Toolbar>
-                <Button style={{left: '0%', marginRight: '5%', color: 'black'}} edge="start" color="inherit" aria-label="Menu">
-                    <MenuIcon />
-                </Button>
-                <Typography variant="h5">
-                    Wisconsin Decarceration Platform
-                </Typography>
-                <Button style={{position: 'absolute', right: '2.5%', color: 'black'}} component={Link} to={ROUTES.SIGN_IN} contained="true" color="inherit"><b>Login</b></Button>
-            </Toolbar>
-        </AppBar>
-    </div>
-);
+    // Control appearance of menu on phone-size screens
+    constructor(props) {
+        super(props)
+        this.state = {
+            toggle: 0
+        }
+    }
 
-export default Navigation;
\ No newline at end of file
+    render() {
+
+        let path = this.props.location.pathname;
+
+        // If there are specific page links use those, else use defaults
+        var links = Object.keys(LINKS.PAGE_LINKS).indexOf(path) == -1 ? LINKS.DEFAULTS : LINKS.PAGE_LINKS[path]
+  
+        return (
+            <div className="nav">
+                <MenuIcon onClick={()=>this.setState({toggle: 1 - this.state.toggle})}/>
+                <ul className={this.state.toggle? "show": ""} onMouseLeave={()=> this.setState({toggle: 0})}>
+                    {links.map(link => (
+                        <li>
+                            {link[0] != path &&

```

Comment:
Use !== instead of ==?

Reply:
This needs to be done for the build to work and be deployed since all warnings prevent compilation for our build environment.

I've been trying to turn this process variable to false but nothing is working so and I think !== is always preferred to != in JavaScript @rs6713 . Thanks!